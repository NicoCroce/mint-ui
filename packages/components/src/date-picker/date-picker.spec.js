import * as React from "react";
import { render, cleanup } from "@testing-library/react";
import DatePicker from "./";

export const TestDatePicker = () => {
	
	return (
		<DatePicker />
	);
};

afterEach(cleanup);

describe("DatePicker tests", () => {
	test("renders correcty", () => {
		const { asFragment } = render(<TestDatePicker />);
		expect(asFragment()).toMatchSnapshot();
	});
	
});
