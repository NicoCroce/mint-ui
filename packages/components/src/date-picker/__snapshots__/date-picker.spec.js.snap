// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RangeSelector tests renders correcty 1`] = `
<DocumentFragment>
  <div
    class="react-datepicker-wrapper"
  >
    <div
      class="react-datepicker__input-container"
    >
      <input
        class=""
        type="text"
        value="04/10/2020"
      />
    </div>
  </div>
</DocumentFragment>
`;
